docs/CODEOWNERS

```diff
@@ -46,7 +46,7 @@ CMakeLists.txt @andschwa @BRMcLaren @EmilAlexandruStoica
 /docs/DesignDocs/ @openenclave/committers
 
 # And governance documents.
-/docs/Contributing.md @openenclave/maintainers
-/docs/GovernanceModel.md @openenclave/maintainers
-/docs/Maintainers.md @openenclave/maintainers
-/docs/Releasing.md @openenclave/maintainers
```

Comment:
can we create a /process subdirectory under /docs to group these and just put openenclave/committee on the whole subdirectory?  We may have other docs over time that get added so want to reduce opportunities for errors.

Reply:
I don't think that'll work for Contributing.md as GitHub specifically references it from docs.

Reply:
(And then I think it would be weird to have them in different places.)

Reply:
In my opinion...not necessary 😉